lib/permutations.rb

```diff
@@ -1,4 +1,23 @@
+# Time Complexity: O(n)
+# Space Complexity: O(n)
 
 def permutations?(string1, string2)
```

Comment:
This works only if the count of each letter is the same.  It returns true however for `heelo` and `hello`.